manifests/kustomize/base/metadata/options/istio/virtual-service.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: metadata-grpc
+  namespace: kubeflow
+spec:
+  gateways:
+  - kubeflow-gateway
+  hosts:
+  - '*'
+  http:
+  - match:
+    - uri:
+        prefix: /ml_metadata
+    rewrite:
+      uri: /ml_metadata
+    route:
+    - destination:
+        host: metadata-envoy-service.kubeflow.svc.cluster.local
```

Comment:
note: this is changed from `host: ml-pipeline-ui.$(kfp-namespace).svc.cluster.local`, because directly routing to metadata-envoy-service reduces one network hop.